.circleci/config.yml

```diff
@@ -112,11 +112,10 @@ jobs:
             python3.4 get-pip.py --user
             export PATH=~/.local/bin:$PATH
             pip install awscli --upgrade --user
-            docker build -t wellcome/wellcomecollection:$CIRCLE_BUILD_NUM .
             docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
-            docker push wellcome/wellcomecollection:$CIRCLE_BUILD_NUM
-            pushd nginx-proxy && ./docker-build.sh $CIRCLE_BUILD_NUM && ./docker-push.sh $CIRCLE_BUILD_NUM && popd
-            pushd router && ./docker-build.sh $CIRCLE_BUILD_NUM && ./docker-push.sh $CIRCLE_BUILD_NUM && popd
+            python ./.circleci/deploy_and_build_docker.py './' 'wellcome/wellcomecollection' $CIRCLE_BUILD_NUM
```

Comment:
Why not run this in a container? Removing concerns about python versions, etc.

Reply:
I am in a container here?

Reply:
https://github.com/wellcometrust/wellcomecollection.org/pull/1747/files/bf82ee9e2ffc7a9b901d31030de6c054001cb5aa#diff-1d37e48f9ceff6d8030570cd36286a61L84